charts/kyverno-notation-aws/values.yaml

```
@@ -76,3 +76,8 @@ tolerations: []
 # affinity for extension Deployment. More details:
 # https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
 affinity: {}
+
+# user supplied labels to apply to all resources excluding label selectors
+commonLabels: {}
```

We should update this to "custom" instead of "common" for more clarity (also this would be aligned with other kyverno helm chart implementations, causing less cognitive load going from one chart to the other 😅 )

```suggestion
customLabels: {}
```